tinync/README.md

```diff
@@ -30,22 +30,29 @@ static void cr_proto(stdin_loop, byte_queue_t *out)
     cr_local uint8_t b;
     cr_local int r;
 
-    cr_begin(); // Initiates the context of this coroutine.
+    /* Initiates the context of this coroutine. */
+    cr_begin();
     for (;;) {
-        cr_sys(r = read(STDIN_FILENO, &b, 1)); // Wait for read system call to be success.
+        /* Wait for read system call to be success. */
+        cr_sys(r = read(STDIN_FILENO, &b, 1));
         if (r == 0) {
-            cr_wait(cr_queue_empty(out)); // Wait until queue out is flushed.
-            cr_exit(1); // Exit the coroutine with status update as finished.
+            /* Wait until queue out is flushed. */
+            cr_wait(cr_queue_empty(out));
+
+            /* Exit the coroutine with status update as finished. */
+            cr_exit(1);
         }
-        cr_wait(!cr_queue_full(out)); // Wait until there is place in queue out.
+        /* Wait until there is place in queue out. */
+        cr_wait(!cr_queue_full(out));
         cr_queue_push(out, b);
     }
-    cr_end(); // End up this coroutine, status will be updated as finished.
+    /* End up this coroutine, status will be updated as finished. */
+    cr_end();
 }
 ```
 
 ### Coroutine Context
-There is another important part of coroutine, that is, context. Context of a coroutine will preserve its execution point which could be resumed later. To define a context for a coroutine, use `cr_context` macro and initiates it with macro `cr_context_init`. It is important to **assign an identical name to context and its corresponding function**. With example presented at the beginning, its corresponding context should be specified as follows:
+Context is an important part for coroutine, which will preserve the execution point of a coroutine that could be resumed later. To define a context for a coroutine, use `cr_context` macro and initiates it with macro `cr_context_init`. It is important to **assign an identical name to context and its corresponding function**. With example presented at the beginning, its corresponding context should be specified as follows:
```

Comment:
The statement "which will preserve" is Inconsistent tense. Change it to something like "which preserves."